CHANGELOG.md

```diff
@@ -175,6 +175,7 @@ New deprecation(s):
 - **CPU scaler**: Wait for metrics window during CPU scaler tests ([#5294](https://github.com/kedacore/keda/pull/5294))
 - **Hashicorp Vault**: Improve test coverage in `pkg/scaling/resolver/hashicorpvault_handler`  ([#5195](https://github.com/kedacore/keda/issues/5195))
 - **Kafka Scaler**: Add more test cases for large value of LagThreshold ([#5354](https://github.com/kedacore/keda/issues/5354))
+- **Kafka Scaler**: Improve readability of utility function getParameterFromConfigV2 ([#5037](https://github.com/kedacore/keda/issues/5037))
```

Comment:
nit: can you please move this to the section for the latest release? This part of the changelog belongs to the `2.13.0` which has been already released and is not going to get new patches.

Reply:
Thanks for the feedback, I just moved it to the latest release.